Source/Diagnostics/BTDiagnostics.H

```diff
@@ -14,47 +14,88 @@ private:
     /** Read relevant parameters for BTD */
     void ReadParameters ();
     /** Flush m_mf_output and particles to file. */
-    // Currently this is an empty function. When OpenPMD/Plotfile is supported
-    // this function can be moved to the base class Diagnostics.
-    void Flush (int i_buffer) override {}
+    // Currently, a temporory customized output format for the buffer
```

Comment:
spelling : temporary

Reply:
Thank you for catching this. I fixed it now.